webpack.rules.js

```diff
@@ -1,32 +1,50 @@
-module.exports = [
+module.exports = rules = [
   // Add support for native node modules
   {
     test: /\.node$/,
-    use: 'node-loader',
-  },
-  {
-    test: /\.(js|jsx|ts|tsx)$/,
```

Comment:
I think it was a proper setup. But you changed it to only `test: /\.tsx?$/,` on line 19